File: InterviewQuestions/Python Interview Programming Questions/task6.py
# Write a Python program to given an array contains prime numbers from 0 to 100

prime_numbers = []

for x in range(0, 101):

    if (x == 0) or (x == 1): continue

    else:

        is_prime = True

        for y in range(2, x): 

            if (x % y == 0): is_prime = False

        if is_prime == True: prime_numbers.append(x)


print(prime_numbers)

File: InterviewQuestions/Javascript Interview Programming Questions/task6.js
// Write a JavaScript program to given an array contains prime numbers from 0 to 100

let prime_numbers = []

for (let i = 0; i <= 100; i ++) {
    
    if (i === 0 || i === 1) continue
    else {

        let is_prime = true
        
        for (let y = 2; y < i; y ++) {

            if (i % y === 0) {

                is_prime = false

                break
            }
        }

        if (is_prime === true) prime_numbers.push(i)
    }
}

console.log(prime_numbers)